docroot/modules/custom/mass_hierarchy/tests/src/ExistingSiteJavascript/HierarchyTest.php

```diff
@@ -118,27 +118,4 @@ public function testCronErasesOtherRevisions() {
     $this->assertSession()->pageTextContains($child1Node->label());
   }
 
-  /**
-   * Tests node edit erases other revisions other than the current.
-   */
-  public function testNodeEditFormErasesOtherRevisions() {
-    $this->drupalLogin($this->createRandomUser('content_team'));
```

Comment:
Why is this hierarchy test removed? I know the form alter is no longer needed but we still want to test that our queue worker does its just, and removed unneeded versions. Is that tested elsewhere?